raiden-dapp/tests/unit/components/amount-input.spec.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Keep it like that. But I think ignoring a whole file is just a last resort. If the linting tool supports it (what it does here) I would prefer to ignore specific lines. Thereby we can explicitly know that the developer ignored this line on purpose. This suppresses all future warnings that could occur without getting noticed. In general this is not too much of an issue. And when the whole files is full of this pattern this also makes sense. So I think for many files you did the best decision.

This comment is meant to share different views/opinions. Please close it after reading or feel free to update the conversation with further arguments form your side.

Reply:
I agree with you, but mostly on src files. For tests, my policy was to put it file-wide if it was more than 2 or 3 instances. Either way, using `any` here is a workaround, hopefully soon enough we'll have better tooling and will be able to remove most of these. Thanks for the headsup!